### ISSUE TITLE
Authorized Applications scopes are hard to parse

### ISSUE THREAD

**Issue by Dorthu** [linode/manager#1639]
Looking at the applications I've granted access to my account, it's hard to see what they can and can't do at a glance.  The strikethrough is classy, but maybe make the struck-through text a lighter grey or bold the other text or add some color.  This was alarming to see at first:

![screen shot 2017-04-25 at 2 43 09 pm](https://cloud.githubusercontent.com/assets/7172953/25402086/b26369ee-29c5-11e7-8326-d259828828dd.png)

**Comment by eatonphil** [comment]
Good point. We should address this. Thank you!
